docs/research/wallet-comparison.md

```diff
@@ -0,0 +1,177 @@
+  - [Zeus](https://zeusln.app/)
+    - Can connect to LND or C-Lightning
+  - [FullyNoded](https://fullynoded.app/)
+    - Can connect to LND or C-Lightning
+- Full nodes
+  - [Breez](https://breez.technology/)
+    - Uses a forked LND version
+  - [Phoenix](https://github.com/ACINQ/phoenix)
+    - Uses a custom lighting implementation
+
+Below we have a look at these wallets and their feature set.
+If a wallet is not mentioned below it has not been analysed (yet).
+
+### Phoenix Wallet
+
+In a nutshell: Non-custodial setup and a great UX that comes with some (minimal) trust for certain features.
+
+Setup:
+
+- Non-custodial setup
+  - Run own LND node on Phone ([Eclair](https://github.com/ACINQ/eclair), ACINQ in-house lightning implementation)
+  - Channel management through ACINQ nodes
+  - Can configure own Electrum Server
+  - Cannot configure own Lightning node (outside the phone setup)
+- Where is trust involved?
+  - Initial channel opening
+  - Onboarding with BTC
+
+Pro:
+
+- Good looking and efficient UI and UX
+- One of the best onboarding UX out there
+  - One-click wallet setup
+  - Onboarding using BTC
+- BIP39 (+BIP84) 12-word wallet backup
+
+Pro/Con:
+
+- Channels management by ACINQ
+  - Slightly higher costs for more convenience
+- Channel backup managed by ACINQ
+  - "Minimal" trust for more convenience
+
+Con:
+
+- Hard to understand the underlying constraints for the Lightning setup
+- Costs for transactions are not transparent (retrospectively)
+  - Example
+    - Initially funded the wallet via BTC, had BTC worth `50.3` USD incoming.
+    - Once they were actually confirmed it changed to `49.95` USD because the fees were deducted.
+    - These fees are never visible again, when going to the details I cannot see how much fee was actually deducted and what for.
+    - A better cost breakdown would be nice to have.
+
+For details check these links:
+
+- https://medium.com/@ACINQ/introducing-phoenix-5c5cc76c7f9e
+- https://phoenix.acinq.co/faq
+
+### Blue Wallet
+
+In a nutshell: Semi- or Non-custodial (when using own LND) setup and a good UX that includes multiple wallet, but overall not as good as Phoenix.
+
+Setup:
+
+- Plug in own LND node, or use [LndHub](https://github.com/BlueWallet/LndHub)
+  - LndHub is a semi-custodial setup ("minimum trust for endusers")
+- Where is trust involved?
+  - LndHub
+
+Pro:
+
+- Multiple wallet support
+- Vaults (Not tried out, feels like a different feature unrelated to Lightning Wallet)
+
+Pro / Con:
+
+- LApp Marketplace
+  - The marketplace is a great way to get an overview of what is out there to use your sats
+  - Does not integrate as deep as Breez, e.g. for LN Markets one can see the app within the app, but one has to register/login separately and cannot just start trading.
+  - Similar to Breez it is unclear what happens when clicking on one of the app icons in the marketplace.
+- One-click wallet create for setup, but onboarding not as clear and rich as Phoenix
```

Comment:
I also like these features

- Plausible deniability: This is a very nice security feature, protecting your money - even if you are physically forced.
- Watch only: I like that as well for security reasons, as I can keep my private keys on my hardware wallet. And I like to check how my BTC is doing ;)